PDSA_System/Client/Models/Admin.cs

```diff
@@ -0,0 +1,17 @@
+namespace PDSA_System.Client.Models
+{
+    public class Admin : Bruker
+    {
+        /*
+        Hvilke andre atributter trenger en admin.
+
+        Rolle
+
+        */
+
+        private Bruker LeggTilNyBruker()
+        {
+            return new Bruker();
+        }
+    }
+}
```

Comment:
Erstatte denne filen med vanlig `Bruker.cs` med adminrolle?

NB: Bør ikke committe direkte i review, for å unngå tomme filer. Slett heller filen i en ny commit
```suggestion

```